### PR TITLE
make tinned foods use openable

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
instead of spawning a seperate item on damage/use

also cleaned it up a lot

spritewise tins now have their own rsis everything is much cleaner
also gave tinned meat its actual source link since the one it used to have didnt even have the sprites there at all!!!!!!

opened maint peaches prototype remains since a map uses it, also gave it a suffix to be less confusing

depends on #19750

## Why / Balance
good

## Technical details
added `OpenBehavior` to replace functionality for breaking open tins of food

for yaml added `FoodOpenableBase` which `FoodTinBase` uses and made all the stuff better

## Media

https://github.com/space-wizards/space-station-14/assets/39013340/d5fa11ae-59c1-441e-b51c-4b0297370a19

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
no cl no fun

### CODE REVIEW

**Comment by Partmedia** [comment]
Is this still intended to be a draft?

**Comment by deltanedas** [comment]
nop

File: Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml [hunk 1]
```diff
@@ -8,7 +8,6 @@
   - type: FlavorProfile
     flavors:
       - meaty
-  - type: Food
```
Comment (Partmedia) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml:11]: What is this doing here?

File: Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml [hunk 2]
```diff
@@ -600,12 +599,11 @@
 # Cooked
 
 - type: entity
-  parent: BaseItem
```
Comment (Partmedia) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml:603]: Unrelated change?

File: Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml [hunk 3]
```diff
@@ -600,12 +599,11 @@
...
+  parent: FoodBase
   id: MaterialSmileExtract
   name: smile extract
   description: It's a real panacea. But at what cost?
   components:
-  - type: Food
```
Comment (Partmedia) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml:608]: Here...

**Review by Partmedia — CHANGES_REQUESTED** [review]
There seem to be some unrelated changes, could you take a look and remove them if appropriate?

File: Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml:11]: unused component

Comment (Partmedia) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml:11]: Please review the [pull request guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). If we find out that these components are actually used, we'd have to revert the whole PR.

Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml:11]: it inherits basefood which provides foodcomponent
obviously its not used since it doesnt have any fields

**Review by mirrorcult — APPROVED** [review]
